team/end-of-iteration-demos.md

```diff
@@ -0,0 +1,10 @@
+---
+title: End of iteration demos
+description: Time to celebrate the great work completed at the end of the iteration
+---
+
+Our teams demo session is an hour on the last day of the iteration. This is a chance to share and celebrate the amazing work done during the iteration. The purpose of this meeting is to give some time to look back at the work completed and to share it with the rest of the team. It's key that everyone is aware of what each other are working on so we don't design and develop in silos.
+
+## Demos completed work
+
+A demo should only be completed work or something you are confident will be complete by the end of the iteration. As when carried forward to the next iteration it will be tempting to demo again. If you are unsure refer to the [backlog sheet](https://docs.google.com/spreadsheets/d/1HH7Elz5ayh4sjH6avh0Z8_4riqlJsZ9JNMxxlesH8Go/edit) for a list of completed epics.
```

Comment:
```suggestion
A squad should only demo completed work or work you are confident will be complete by the end of the iteration. This is because it is too tempting to share the work again in the next demo.

 If you are unsure refer to the [backlog sheet](https://docs.google.com/spreadsheets/d/1HH7Elz5ayh4sjH6avh0Z8_4riqlJsZ9JNMxxlesH8Go/edit) for a list of completed epics and ask other people on your squad for advice.
```